code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -289,12 +290,16 @@ mob/living/carbon/bullet_act(obj/item/projectile/P, def_zone)
 
 
 /mob/living/carbon/damage_clothes(damage_amount, damage_type = BRUTE, damage_flag = 0, def_zone)
+	if(damage_type != BRUTE && damage_type != BURN)
+		return
 	var/bodypart_bit = 0
+	if(damage_flag != "bomb")

```

Comment:
there are no defines for most of these damage types. It's gross. I've been intending to rectify that when i do my armor refactor.